README.md

```diff
@@ -297,7 +302,10 @@ A representation of a client. _Inherits from EventEmitter_.
       sending binary data, which is sent as is.
     - **Parameters**
       - `String` | `Buffer` | `ArrayBuffer` | `ArrayBufferView`: a string or any object implementing `toString()`, with outgoing data, or a Buffer or ArrayBuffer with binary data. Also any ArrayBufferView can be sent as is.
+      - `Object`: optional, options object
       - `Function`: optional, a callback executed when the message gets flushed out by the transport
+    - **Options**
+      - `compress` (`Boolean`): whether to compress sending data. This option might be ignored and forced to be `true` when using polling. (`true`)
```

Comment:
Umm, I'm not sure which is better, but thought ignoring the option would be faster since we need a poll request per packet in the worst case if we don't do so.